tedana/decomposition/ica.py

```diff
@@ -67,7 +67,7 @@ def tedica(
 
     ica_method = ica_method.lower()
 
-    if ica_method == "robustica":
+    if ica_method == "robustica":    ##The issue with fixed seed in robustica was resolved, see https://github.com/CRG-CNAG/robustica/issues/3
```

Comment:
I know I mentioned that there should be some kind of reference to the resolution of the fixed seed in robustica issue, but this doesn't work. If you look at the code naive to this history, it's a very odd code comment that seemingly bears no relevance to the surrounding code.

What I was advocating was something like, for instance, if it was the case that you perhaps made some code change where previously you were forbidding use of a fixed seed with robustica but it was now permitted, perhaps with an increment to the requisite robustica version number, then the commit making that change should include in its commit message a description of and hyperlink to that upstream change.